auth0/v3/management/grants.py

```diff
@@ -0,0 +1,63 @@
+from .rest import RestClient
+
+
+class Grants(object):
+
+    """Auth0 grants endpoints
+
+    Args:
+        domain (str): Your Auth0 domain, e.g: 'username.auth0.com'
+
+        token (str): Management API v2 Token
+
+        telemetry (bool, optional): Enable or disable Telemetry
+            (defaults to True)
+    """
+
+    def __init__(self, domain, token, telemetry=True):
+        self.domain = domain
+        self.client = RestClient(jwt=token, telemetry=telemetry)
+
+    def _url(self, id=None):
+        url = 'https://%s/api/v2/grants' % self.domain
+        if id is not None:
+            return url + '/' + id
+        return url
+
+    def all(self, page=None, per_page=None, include_totals=False, extra_params=None):
+        """Retrieves all grants.
+
+        Args:
+            page (int, optional): The result's page number (zero based).
+
+            per_page (int, optional): The amount of entries per page.
+
+            include_totals (bool, optional): True if the query summary is
+                to be included in the result, False otherwise.
+
+           extra_params (dictionary, optional): The extra parameters to add to
+             the request. The fields, include_fields, page and per_page values
```

Comment:
`The fields, include_fields, page and per_page values` should be `The page, per_page and include_totals values`. Probably copy paste issue.

```suggestion
             the request. The page, per_page and include_totals values
```